Composer/packages/server/src/externalContentProvider/powerVirtualAgentsProvider.ts

```diff
@@ -158,7 +160,8 @@ export class PowerVirtualAgentsProvider extends ExternalContentProvider<PowerVir
 
   private getContentUrl(): string {
     const { envId, baseUrl, botId } = this.metadata;
-    return `${baseUrl || getBaseUrl()}/environments/${envId}/bots/${botId}/composer/content`;
+    const query = process.env.COMPOSER_PVA_TOPICS === 'true' ? '?includeTopics=true' : '';
```

Comment:
Where does this `COMPOSER_PVA_TOPICS` variable get set? Are we going to turn this on at some point in production or is there somewhere we set it that I'm not seeing?

Reply:
It's not set by default at the moment. I figure we'd default it to true whenever the PVA changes make it to prod.

Reply:
Sounds good